mantidimaging/gui/widgets/mi_image_view/view.py

```diff
@@ -130,7 +130,7 @@ def histogram(self) -> HistogramLUTItem:
 
     @property
     def image_data(self) -> np.ndarray:
-        return self.image
+        return self.image if self.image is not None else np.array([])
```

Comment:
I think it would be better to change the type signature to `np.ndarray|None` than the behaviour.

I found `ImageViewLineROI._image_data_exists()`, uses this to check if there is an image.

Reply:
I can change the type. signature only if I also change the type signature of `image_data` in `AutoColorMenu` otherwise `image_data` will be incompatible with the supertype `AutoColorMenu`

Reply:
I think that's the right thing to do.